tools/bridge/bridge/confirmation_sender.py

```diff
@@ -0,0 +1,105 @@
+import logging
+from typing import Any, Dict
+
+import gevent
+from gevent.queue import Queue
+
+from web3.contract import Contract
+from eth_keys.datatypes import PrivateKey
+from eth_utils import keccak, int_to_big_endian
+
+from bridge.constants import HOME_CHAIN_STEP_INTERVAL
+
+
+class ConfirmationSender:
+    """Sends confirmTransfer transactions to the home bridge contract."""
+
+    def __init__(
+        self,
+        transfer_queue: Queue,
+        home_bridge_contract: Contract,
+        private_key: bytes,
+        gas_price: int,
+        max_reorg_depth: int,
+    ):
+        self.transfer_queue = transfer_queue
```

Comment:
Suggestion: I start to feel the need to distinguish between a token transfer and a bridge transfer. I think it is not much work to add a prefix where it fits, but could help everyone new digging into the code. Even us it can help to don't get confused. Probably we need to add this at other placed as well.